exchanges/kraken/testdata/wsOpenTrades.json

```diff
@@ -0,0 +1,6 @@
+[[{"OGTT3Y-C6I3P-XRI6HR":{"cost":"0.00000","descr":{"close":"","leverage":"0.1","order":"sell 10.00345345 XBT/USD @ limit 34.50000 with 0:1 leverage","ordertype":"limit","pair":"XBT/USD","price":"34.50000","price2":"0.00000","type":"sell"},"expiretm":"0.000000","fee":"0.00000","limitprice":"34.50000","misc":"","oflags":"fcib","opentm":"0.000000","avg_price":"0.00000","refid":"LIMIT-OPEN","starttm":"0.000000","status":"open","stopprice":"0.000000","userref":0,"vol":"10.00345345","vol_exec":"0.00000000"}},{"OKB55A-UEMMN-YUXM2A":{"avg_price":"0.00000","cost":"0.00000","descr":{"close":null,"leverage":null,"order":"buy 0.00010000 XBT/USDT @ market 0.00000","ordertype":"market","pair":"XBT/USDT","price":"0.00000","price2":"0.00000","type":"buy"},"expiretm":null,"fee":"0.00000","limitprice":"0.00000","misc":"","oflags":"fciq","opentm":"1692851641.361371","refid":null,"starttm":null,"status":"pending","stopprice":"0.00000","timeinforce":"GTC","userref":0,"vol":"0.00010000","vol_exec":"0.00000000"}}],"openOrders",{"sequence":1}]
```

Comment:
Since the data in it is json, and parsable, it felt right to call it json.
I'd like to keep it as json, unless I'm missing something. Is there an issue with it?

I checked out the top level testdata directory, and it didn't feel right for this.
I also checked for best practices again to be sure, and fixtures like this usually live in their package directory. Colocating them together makes it less likely they'll be forgotten in refactoring, removal or abstraction

Reply:
The data contained in the file is not technically JSON and you are not parsing it as a complete JSON object, as such makes the IDE wiggout, but it's not a big issue. As the file itself location, that makes sense. You just have an option to consolidate if you so choose.

Reply:
Okay, understood.
I'll choose to leave as is, then.
It's a file of json objects, but not parsable as an entire document.
If it *really* breaks the IDE you're using (which is ...? ) then I'm happy to move to .txt

Reply:
It doesn't break it, just complains, VSCode:

<img width="272" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/a9bbf5f1-2b8a-4c1b-ba9c-99bbc11f58ee">